src/Post/Post.js

```diff
@@ -10,6 +8,41 @@ import { useIGData } from '../hooks/useIGData';
 import { absoluteToRelativeDate } from '../utils';
 import './Post.css';
 
+function PostLikeButton({
+  is_post_liked,
+  index,
+  toggleLike,
+  commentId,
+  replyId,
+  size = '24',
+  width = '28',
+  isMarginNeeded,
+}) {
+  return (
+    <>
+      <button
+        className="post-like-button"
+        onClick={() => toggleLike(index, commentId, replyId)}
```

Comment:
If you want to implement my suggestion from above, all you have to do is
```suggestion
        onClick={onClick}
```